This patch changes Qremote to be a bit more tolerant about the greeting
of the remote server. RfC 2821, section 4.2 specifies that a greeting begins
with "220 ", so multiline greetings are forbidden. Nevertheless there are
some hosts doing this, normally Qremote will refuse to send a mail to them.

Index: qremote/qremote.c
===================================================================
--- qremote/qremote.c	(revision 519)
+++ qremote/qremote.c	(working copy)
@@ -431,6 +431,10 @@
 			quitmsg();
 			continue;
 		}
+		while (!strncmp("220-", linein, 4)) {
+			if (netget() != 220)
+				break;
+		}
 		if (linein[3] != ' ') {
 			const char *dropmsg[] = {"invalid greeting from ", NULL, NULL};
 
